manufacturing-client/src/main.rs

```diff
@@ -296,10 +301,79 @@ async fn get_mfg_info(mfg_string_type: MfgStringType) -> Result<CborSimpleType>
     }
     match mfg_string_type {
         MfgStringType::SerialNumber => {
-            let serial = fs::read_to_string("/sys/devices/virtual/dmi/id/product_serial")
+            log::debug!(
+                "msg_string_type value:{:#?} refers SerialNumber format ",
+                mfg_string_type
+            );
+            let serial_dmi = fs::read_to_string("/sys/devices/virtual/dmi/id/product_serial")
                 .or_else(|_| fs::read_to_string("/sys/devices/virtual/dmi/id/chassis_serial"))
                 .context("Error determining system serial number")?;
-            Ok(CborSimpleType::Text(serial))
+            Ok(CborSimpleType::Text(serial_dmi))
+        }
+        MfgStringType::SerialNumberExt => {
+            log::debug!(
+                "msf_string_type value:{:#?} refers SerialNumberExt format ",
```

Comment:
nit: `s/msf_string_type/msg_string_type/`

Reply:
Or is this really supposed to be `mfg_string_type`?  I'm getting confused by a similar statement on lines 305-306

https://github.com/sarmahaj/fido-device-onboard-rs/blob/343d959b97ec062e306f9d73eb0a527c55e17ec2/manufacturing-client/src/main.rs#L305-L306

I think they should be the same, whichever is correct

Reply:
`mfg` is the shortened version of `manufacturer` <https://fidoalliance.org/specs/FDO/FIDO-Device-Onboard-PS-v1.1-20220419/FIDO-Device-Onboard-PS-v1.1-20220419.html#protocolEnt>, it should be `mfg`